apps/frontend/src/app/5_pages/ConvertPage/hooks/useHandleConversion.ts

```diff
@@ -138,91 +131,52 @@ export const useHandleConversion = (
       return;
     }
 
-    const requiresPermit2 =
-      !!route && ['ZeroRedemption', 'MocIntegration'].includes(route.name);
+    // const requiresPermit2 =
```

Comment:
You can probably just delete it or do you want to leave it here for some specific reason?

Reply:
Ah, no, just forgot to remove it